README.md

```diff
@@ -80,6 +80,12 @@ There is already [PythonInMusic](https://wiki.python.org/moin/PythonInMusic) but
 * [SIDEKIT](http://lium.univ-lemans.fr/sidekit/)³ - Speaker and Language recognition
 * [SpeechRecognition](https://github.com/Uberi/speech_recognition) -  Wrapper for several ASR engines and APIs, online and offline
 * [talkbox](http://scikits.appspot.com/talkbox)² - General speech/signal processing algorithms. ⚠️ Not maintained
+* [aeneas](https://www.readbeyond.it/aeneas/) - Forced aligner, based on MFCC+DTW, 35+ languages
+* [FAVE-align](https://github.com/JoFrhwld/FAVE/) - Forced aligner, based on HTK (HMM), English. ⚠️ Not on PyPI
+* [Gentle](https://lowerquality.com/gentle/) - Forced aligner, based on Kaldi (HMM), English. ⚠️ Not on PyPI
+* [Montreal Forced Aligner](https://montrealcorpustools.github.io/Montreal-Forced-Aligner/) - Forced aligner, based on Kaldi (HMM), English (others can be trained). ⚠️ Not on PyPI
+* [Prosodylab-Aligner](http://prosodylab.org/tools/aligner/)³ - Forced aligner, based on HTK (HMM), English (others can be trained). ⚠️ Not on PyPI
+* [SPPAS](http://www.sppas.org/index.html)² - Forced aligner, based on Julius (HMM), 12 languages (others can be trained). ⚠️ Not on PyPI
```

Comment:
I have some concerns about SPPAS:

1. it seems more like end-user software including a UI and there seems to be no way to use the algorithm from outside the software.
2. the source code is not on github or any other code repository, so it seems that this is not an open source project where contributions are welcome
3. it seems difficult to install